apps/playground/src/app/modules/components/button-module/button-module.component.ts

```diff
@@ -2,25 +2,68 @@
  * Angular imports
  */
 import { Component, OnInit } from '@angular/core';
-import { FormBuilder, FormGroup } from '@angular/forms';
+import { FormControl, FormGroup, NonNullableFormBuilder, Validators } from '@angular/forms';
+
+/**
+ * 3rd-party imports
+ */
+import { ElementAlignment, ElementSize } from '@betagouv/ngx-dsfr';
+import { ButtonHtmlType, ButtonType } from '@betagouv/ngx-dsfr/button';
 
```

Comment:
I have mistakes in the Template because Typed Forms is not correctly implemented.

You should add this here :
```typescript
/**
 * TypeScript entities and constants
 */
interface ButtonForm {
  label: FormControl<string>;
  title: FormControl<string>;
  disabled: FormControl<boolean>;
  size: FormControl<ElementSize>;
  htmlType: FormControl<ButtonHtmlType>;
  type: FormControl<ButtonType>;
  icon: FormControl<string>;
  iconAlignment: FormControl<ElementAlignment>;
}
```